### ISSUE TITLE
[REQUEST] Option to display the output dimensions of the layers

### ISSUE THREAD

**Issue by JoeSykesNHS** [paulgavrikov/visualkeras#43]
**Describe the solution you'd like**
It would be nice if the graph could display the output sizes of the layers. This would then mean that the architecture would not need to also be saved using `plot_model` from `keras.utils`. 

**Describe alternatives you've considered**
The current alternative is to save the graph produced by `visualkeras` and also an architecture diagram produced by `plot_model`.

**Additional context**
something similar to the image below, but just the dimension sizes.
![image](https://github.com/paulgavrikov/visualkeras/assets/127425466/129e5f6f-9478-4f82-995e-51268de3366b)

**Comment by thegodone** [comment]
Is this is possible now ? and how ?

**Comment by Soontosh** [comment]
@thegodone Yes, this is possible now. Take a look at the README for more information: https://github.com/paulgavrikov/visualkeras?tab=readme-ov-file#drawing-information-text

Doing this requires you to pass in a custom-made callable function. You should configure this function to return whatever information (as a string) that you'd like to be displayed under (or above) each layer.

You'll probably want to play around with the font, padding (mainly those two arguments, but others as well if needed) to get a professional looking output image.

Let me know if you have any more questions.
